torch/onnx/_internal/fx/export.py

```diff
@@ -187,118 +274,89 @@ def _export_fx_to_ts(fx_module_with_metadata, opset_version):
+
+            fx_name_to_onnxscipt_tensor[node.name] = output
         elif node.op == "call_function":
             # aten ops and other statless functions.
             if (
                 isinstance(node.target, torch._ops.OpOverload)
                 and node.target in _OP_OVERLOAD_TO_EXPORTER_KEY_TABLE
             ):
                 exporter_key = _OP_OVERLOAD_TO_EXPORTER_KEY_TABLE[node.target]
-                symbolic_function_group = registration.registry.get_function_group(
-                    exporter_key
-                )
-                assert symbolic_function_group is not None
-                symbolic_fn = symbolic_function_group.get(opset_version)
-                assert symbolic_fn is not None
-                # TODO(wechi): current type checking throws when feeding torch._C.Graph
-                # to symbolic_opset*.py functions, so we need the following wrapper.
-                # After we get rid of TorchScript, we can remove this wrapper.
-                graph_context = jit_utils.GraphContext(
-                    graph=g,
-                    block=g.block(),  # Pointless. Just make linter happy.
-                    opset=opset_version,
-                    original_node=g.insertPoint(),  # Pointless. Just make linter happy.
-                    params_dict={},  # Pointless. Just make linter happy.
-                    env={},  # Pointless. Just make linter happy.
-                )
+
+                # only latest opset version is only supported in atenlib for now
+                symbolic_fn = _ATENLIB_FUNCTIONS.get(exporter_key)
+                assert (
+                    symbolic_fn is not None
+                ), f"Cannot find function for {exporter_key}"
                 # Map FX inputs to ONNX inputs and fill optional inputs with default values.
-                ts_args = _wrap_fx_args_as_ts_args(
-                    graph_context, fx_module_with_metadata, node, fx_name_to_ts_value
+                onnx_args = _wrap_fx_args_as_ts_args(
+                    fx_module_with_metadata, node, fx_name_to_onnxscipt_tensor
                 )
-                # The returned value could be a value of a tuple of values.
-                v = symbolic_fn(graph_context, *ts_args)
+                with evaluator.default_as(tracer):
+                    output: Union[
+                        graph_building.TorchScriptTensor,
+                        Tuple[graph_building.TorchScriptTensor],
+                    ] = symbolic_fn(*onnx_args)
                 assert (
-                    v is not None
-                ), f"Node creates None with target={node.target}, name={node.name}, args={ts_args}"
+                    output is not None
+                ), f"Node creates None with target={node.target}, name={node.name}, args={onnx_args}"
                 # Assign type and shape obtained from FakeTensorProp.
                 # _fill_tensor_types(v, node.meta["val"])
```

Comment:
I am not sure if this is required. If ONNXScript has its own shape inference logic, this can be removed. Be sure to test the exporting of GPT abd bloom before merging this PR.

Reply:
I think we do want this. Will update